vue/src/modules/Workspace/WorkspaceView.vue

```diff
@@ -54,19 +54,25 @@ export default {
   data() {
     return {
       breadcrumbs: [],
-      items: [
-        {title: this.$i18n.t('Timeline'), to: {name: 'workspace.timeline'}, icon: 'fa fa-bars'},
-        {title: this.$i18n.t('Files'), to: {name: 'workspace.files'}, icon: 'fa fa-folder'},
-        {title: this.$i18n.t('Articles'), to: {name: 'workspace.articles'}, icon: 'fa fa-book'},
-        {title: this.$i18n.t('About'), to: {name: 'workspace.about'}, icon: 'fa fa-info-circle'},
-      ]
     }
   },
   computed: {
     ...mapState({
       workspace: state => state.view.workspace,
       loading: state => state.view.loading,
     }),
+    items() {
+      return [
+        {title: this.$i18n.t('Timeline'), to: {name: 'workspace.timeline'}, icon: 'fa fa-bars'},
+        {
+          title: this.$i18n.t('Files'),
+          to: {name: 'workspace.files', params: {folderId: this.workspace.rootFolders.id}},
```

Comment:
@skavt Workspace should have only 1 root folder, therefore the attribute name should be `this.workspace.rootFolder` instead of `this.workspace.rootFolders`